gips/data/landsat/landsat.py

```diff
@@ -807,6 +807,17 @@ def filter(self, pclouds=100, sensors=None, **kwargs):
                 return False
         return True
 
+    def masks(self, patterns=None):
```

Comment:
I considered this, but due to the late hour, balked.  Something didn't seem right about doing a few things, then calling super...but if that seems cool to you, then I like it.